src/data/modifiers.txt

```diff
@@ -4100,6 +4100,8 @@ Item	Time Cape	Item Drop: [25*zone(Twitch)]
 # Time Cloak: Sometimes Find an Extra Chroner in the Time-Twitching Tower
 Item	Trainbot harness	Initiative: -200
 # toy jet pack: Jane!  Get Me Off This Crazy Thing!
+# Trainbot harness: Lets you rescue elves from passenger-occupied Crimbo train cars
+Item	Trainbot harness	Initiative: -200
```

Comment:
Look at line 4101!

Reply:
Thanks! I will remove it, since it is not in the correct (alphabetic) place.